libs/SFMLSystems/InputManagement.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+** EPITECH PROJECT, 2022
+** rtype
+** File description:
+** InputManagement
+*/
+
+#include "InputManagement.hpp"
+#include "World/World.hpp"
+#include "RenderWindowResource.hpp"
+#include "MouseInputComponent.hpp"
+#include "KeyboardInputComponent.hpp"
+#include "ControllerJoystickInputComponent.hpp"
+#include "ControllerButtonInputComponent.hpp"
+#include "ActionQueueComponent.hpp"
+#include "AllowControllerComponent.hpp"
+#include "AllowMouseAndKeyboardComponent.hpp"
+
+using namespace ecs;
+
+void InputManagement::run(World &world)
+{
+    sf::Event event;
+    std::vector<std::shared_ptr<Entity>> Inputs = world.joinEntities<MouseInputComponent, KeyboardInputComponent, ControllerButtonInputComponent, ControllerJoystickInputComponent, ActionQueueComponent>();
```

Comment:
Beware ! Join Entities isn't an "This component or This one" it's like an AND "This component and This one"

Reply:
I know it, the entity must have all these components.